man/git-get.md

```diff
@@ -0,0 +1,34 @@
+git-get(1) -- Clone a Git repository under a directory
+=================================================
+
+## SYNOPSIS
+
+`git-get`
+
+## DESCRIPTION
+
+  Clones a Git repository under the directory specified by the environment variable `GIT_EXTRA_DEFAULT_CLONE_PATH`
```

Comment:
What about using a git config like the other scripts?
For example,
https://github.com/tj/git-extras/blob/f5bcbe5b0d4f683663da96e7953a151ede8cfaf0/bin/git-pr#L62

Reply:
Yeah that's probably better